tests/ci/release.py

```diff
@@ -170,26 +183,24 @@ def log_rollback(self):
             )
 
     @contextmanager
-    def prestable(self):
+    def create_release_branch(self):
         self.check_no_tags_after()
         # Create release branch
         self.read_version()
         with self._create_branch(self.release_branch, self.release_commit):
             with self._checkout(self.release_branch, True):
                 self.read_version()
-                self.version.with_description(VersionType.PRESTABLE)
-                with self._create_gh_release(True):
-                    with self._bump_prestable_version():
+                self.version.with_description(self.get_stable_release_type())
+                with self._create_gh_release(False):
```

Comment:
Seems `self._create_gh_release(True)` is not called anymore, so can we remove argument?

Reply:
I'll leave it for our `-testing` releases. They will be pre-release releases